__tests__/database/schema/UserSchema.test.ts

```diff
@@ -495,7 +496,8 @@ describe('TESTING UserSchema', () => {
         countryCode: 'countryCode',
         createdAt: 'createdAt',
         lastActiveAt: 'lastActiveAt',
-        profilePicture: 'profilePicture',
+        profilePicture:
+          'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `profilePicture` property has been hardcoded with a URL. If this is a test, it's fine. However, if this code is used in production, it would be better to use a variable or constant instead of hardcoding the URL.
```diff
-        profilePicture:
-          'https://res.cloudinary.com/hpjivutj2/image/upload/v1680929851/default-profile-picture_vrmmdn.png',
+        profilePicture: DEFAULT_PROFILE_PICTURE_URL,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->